### PR TITLE
feat: add substrate-transfer precompile json for Blockscout

### PR DESCRIPTION
**Pull request by DylanVerstraete**
# Description of proposed changes

<describe what this PR is about and why we want it>

---

Practical tips for PR review & merge:

- [ ] All GitHub Actions report PASS
- [ ] Newly added code/functions have unit tests
  - [ ] Coverage tools report all newly added lines as covered
  - [ ] The positive scenario is exercised
  - [ ] Negative scenarios are exercised, e.g. assert on all possible errors
  - [ ] Assert on events triggered if applicable
  - [ ] Assert on changes made to storage if applicable
- [ ] Modified behavior/functions - try to make sure above test items are covered
- [ ] Integration tests are added if applicable/needed

### CODE REVIEW

**Review by beqaabu — DISMISSED** [review]

File: precompiles/metadata/precompiles-creditcoin3-testnet.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+      "abi": "[{\"anonymous\":false,\"inputs\":[{\"indexed\":false,\"internalType\":\"bytes32\",\"name\":\"destination\",\"type\":\"bytes32\"},{\"indexed\":false,\"internalType\":\"uint256\",\"name\":\"amount\",\"type\":\"uint256\"}],\"name\":\"Transfer\",\"type\":\"event\"},{\"inputs\":[{\"internalType\":\"bytes32\",\"name\":\"destination\",\"type\":\"bytes32\"},{\"internalType\":\"uint256\",\"name\":\"amount\",\"type\":\"uint256\"}],\"name\":\"transfer_substrate\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"}]"
+    }
+  ]
+
```
Comment (atodorov) [inline precompiles/metadata/precompiles-creditcoin3-testnet.json:11]: @DylanVerstraete Question: do we need to make sure that substrate_transfer.sol alaways matches  the value in this json file? 

The check can be done as a follow up.

Comment (DylanVerstraete) [reply precompiles/metadata/precompiles-creditcoin3-testnet.json:11]: We can yes

Comment (atodorov) [reply precompiles/metadata/precompiles-creditcoin3-testnet.json:11]: FTR opened CSUB-1169 for this. Closing this comment!

**Review by atodorov — DISMISSED** [review]
LGTM. 

Restarted failing CI jobs b/c the Linode client library was ratelimited b/c there were several commits in quick succession.

**Comment by atodorov** [comment]
pre-commit.ci autofix

**Review by atodorov — APPROVED** [review]

**Review by beqaabu — APPROVED** [review]

**Pull request merged**